flink-sql-runner/src/main/java/com/github/streamshub/flink/SqlRunner.java

```diff
@@ -74,44 +84,101 @@ public static void main(String[] args) throws Exception {
         }
     }
 
-    static List<String> parseStatements(String rawStatements) {
-        var formatted = formatSqlStatements(rawStatements.trim());
+    static List<String> parseStatementArg(String rawStatementArg) {
 
-        var statements = new ArrayList<String>();
-        StringBuilder current = new StringBuilder();
-        Matcher matcher = STATEMENT_SET_PATTERN.matcher(formatted);
+        LOG.debug("Parsing raw statement:\n<begin>{}<end>", rawStatementArg);
 
-        String statementSet = "";
-        String otherStatements = formatted;
+        var statements = new ArrayList<String>();
 
-        if (matcher.find()) {
-            statementSet = matcher.group(1);
-            otherStatements = formatted.replace(statementSet, "").trim();
-        }
+        var cleaned = cleanSqlStatement(rawStatementArg);
+        var formatted = formatSqlStatements(cleaned);
+
+        var statementSetString = new StringBuilder();
```

Comment:
`setStatmentBuilder`? `processedSetStatment` ?

Reply:
Changed it to option A